.github/workflows/codeql-analysis.yml

```diff
@@ -4,12 +4,20 @@ on:
   push:
     branches:
       - main
+      - next
       - release/*
   pull_request:
     # The branches below must be a subset of the branches above
     branches:
       - main
+      - next
```

Comment:
@heliocliu There's no baseline CodeQL run for the next branch yet, but it will be after this PR is merged -- so future merges shouldn't see false positives like this PR did.